pyproject.toml

```diff
@@ -37,6 +37,13 @@ build-backend = "poetry.core.masonry.api"
 [tool.ruff]
 target-version = "py311"
 extend-exclude = [".cache"]
+line-length = 120
+unsafe-fixes = true
+preview = true
+output-format = "concise"
```

Comment:
This was added to make sure it uses the short output by default. Without this `pre-commit run` output is very long